python/sglang/srt/managers/scheduler.py

```diff
@@ -1817,20 +1817,13 @@ def update_running_batch(self, batch: ScheduleBatch) -> Optional[ScheduleBatch]:
             TEST_RETRACT and self.forward_ct % TEST_RETRACT_INTERVAL == 0
         ):
             old_ratio = self.new_token_ratio
-            retracted_reqs, new_token_ratio, reqs_to_abort = batch.retract_decode(
-                self.server_args
-            )
+            retracted_reqs, new_token_ratio = batch.retract_decode(self.server_args)
             self.num_retracted_reqs = len(retracted_reqs)
             self.new_token_ratio = new_token_ratio
-            for req in reqs_to_abort:
-                self.send_to_tokenizer.send_output(
-                    AbortReq(abort_reason=req.to_abort_message, rid=req.rid), req
-                )
```

Comment:
we need this in one case of PD when retract happens and the host runs out of the memory